.github/workflows/integration-test-upgrade-suite.yaml

```
@@ -30,18 +30,66 @@ jobs:
         github-token: ${{ secrets.GITHUB_TOKEN }}
         kubernetes-version: ${{ matrix.kubernetes-versions }}
 
-    - name: TestCephUpgradeSuite
+    - name: TestCephUpgrade
       run: |
         tests/scripts/github-action-helper.sh collect_udev_logs_in_background
         export DEVICE_FILTER=$(lsblk|awk '/14G/ {print $1}'| head -1)
-        go test -v -timeout 1800s -failfast -run CephUpgradeSuite/TestUpgradeRookToMaster github.com/rook/rook/tests/integration
+        go test -v -timeout 1800s -failfast -run CephUpgradeSuite/TestUpgradeRook github.com/rook/rook/tests/integration
 
     - name: collect common logs
       if: always()
       run: |
         export LOG_DIR="/home/runner/work/rook/rook/tests/integration/_output/tests/"
-        export CLUSTER_NAMESPACE="upgrade-ns"
-        export OPERATOR_NAMESPACE="upgrade-ns-system"
+        export CLUSTER_NAMESPACE="upgrade"
+        export OPERATOR_NAMESPACE="upgrade-system"
+        tests/scripts/collect-logs.sh
+
+    - name: Artifact
+      uses: actions/upload-artifact@v2
+      if: failure()
+      with:
+        name: ceph-upgrade-suite-artifact-${{ matrix.kubernetes-versions }}
+        path: /home/runner/work/rook/rook/tests/integration/_output/tests/
+
+    - name: setup tmate session for debugging when event is PR
+      if: failure() && github.event_name == 'pull_request'
+      uses: mxschmitt/action-tmate@v3
+      timeout-minutes: 60
+
+  TestHelmUpgrade:
+    if: ${{ github.event_name == 'pull_request' && github.ref != 'refs/heads/master' && !contains(github.event.pull_request.labels.*.name, 'skip-ci') }}
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        kubernetes-versions : ['v1.16.15', 'v1.23.0']
+    steps:
+    - name: checkout
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: setup cluster resources
+      uses: ./.github/workflows/integration-test-setup
+      with:
+        github-token: ${{ secrets.GITHUB_TOKEN }}
+        kubernetes-version: ${{ matrix.kubernetes-versions }}
+
+    - name: TestHelmUpgrade
```

I see this is named with "Suite" for the on-release tests. Is there a reason to not keep the names consistent?

```suggestion
    - name: TestHelmUpgradeSuite
```

---

The upgrade tests don't really run as a suite, the github actions call individual tests inside the suite for the ceph upgrade, helm upgrade, and the daily tests for ceph pacific-devel upgrade, etc. So it seemed appropriate to remove the "Suite" suffix from the test name.

---

Right, but one ends in Suite and the other doesn't, and I'm not sure if I've missed something or if one of them should be changed.